src/lib/operators/projection.cpp

```diff
@@ -41,9 +41,9 @@ std::shared_ptr<AbstractOperator> Projection::recreate(const std::vector<AllPara
 template <typename T>
 void Projection::_create_column(boost::hana::basic_type<T> type, const std::shared_ptr<Chunk>& chunk,
                                 const ChunkID chunk_id, const std::shared_ptr<PQPExpression>& expression,
-                                std::shared_ptr<const Table> input_table_left) {
+                                std::shared_ptr<const Table> input_table_left, bool only_column_expressions) {
```

Comment:
I don't like the name of this. How about something like `forward_pointer_only`, `reuse_column_from_input` or something else?

Reply:
I like `reuse_column_from_input`. Changed it.